static_src/test/unit/stores/user_store.spec.js

```diff
@@ -134,22 +135,25 @@ describe('UserStore', function () {
       expectedUsers = [
         {
           guid: userGuidA,
-          roles: { [spaceGuid]: ['space_developer'] }
+          roles: { [spaceGuid]: ['space_developer'] },
+          space_roles: [ 'space_developer' ]
```

Comment:
Its not a new attribute, its an old one. `space_roles` is returned when we fetch the user from the API. We do a transformation that adds the `roles` key, but in order to pass a deep equality test, the object needs to retain the `space_roles` key. 

We could edit the code to the delete `space_roles` from the underlying object, but I don't think it hurts to have the extra field.